code/modules/projectiles/guns/ego_gun/waw.dm

```diff
@@ -76,10 +78,17 @@
 	fire_delay = initial(fire_delay)
 	var/mob/living/carbon/human/myman = user
 	var/obj/item/clothing/suit/armor/ego_gear/magicbullet/Y = myman.get_item_by_slot(ITEM_SLOT_OCLOTHING)
+	var/obj/item/clothing/suit/armor/ego_gear/realization/bigiron/Z = myman.get_item_by_slot(ITEM_SLOT_OCLOTHING)
 	if(istype(Y))
 		fire_delay = 15
+	if(istype(Z))
+		projectile_damage_multiplier = 2.5
+	else
+		projectile_damage_multiplier = 1
+	return
```

Comment:
Remove this return, this stops it from firing entirely

Reply:
It worked fine in testing, but sure!